mods/d2k/rules/atreides.yaml

```diff
@@ -1,282 +0,0 @@
-CONYARDA:
-	Inherits: ^CONYARD
-
-PWRA:
-	Inherits: ^POWER
-	Buildable:
-		Prerequisites: ~conyarda
-
-WALLA:
-	Inherits: ^WALL
-	Buildable:
-		Prerequisites: ~conyarda, barracks
-
-GUNTOWERA:
-	Inherits: ^GUNTOWER
-	Buildable:
-		Prerequisites: ~conyarda, barracks
-
-ROCKETTOWERA:
-	Inherits: ^ROCKETTOWER
-	Buildable:
-		Prerequisites: ~conyarda, radar
-
-REFA:
-	Inherits: ^REFINERY
-	Buildable:
-		Prerequisites: ~conyarda, power
-
-BARRA:
```

Comment:
This has two exits at different locations:

![image](https://cloud.githubusercontent.com/assets/756669/6202985/e2da937e-b508-11e4-96da-ceaad0ca99a4.png)

I was about to fix that, but won't be able to if you just remove it.